hydromt/cli/_utils.py

```diff
@@ -85,20 +83,12 @@ def parse_json(ctx, param, value: str) -> Dict[str, Any]:
 
 
 ### general parsing methods ##
-
-
-def parse_config(
+def _parse_config(
```

Comment:
Can you explain how this function is a private function but is still exported via `__all__`? What is the purpose of making this a private function?

Reply:
The exported via `__all__` is my mistake. whether this is a private or a public functions is a bit debatable since it's in a private sub module. This function used to be public, but I think it's a good idea to start clamping down the internal stuff like cli and config parsing a bit more. I just didn't remember to apply it consistently. What do you think? Should this be applied to all functions here or should I revert it to public?

Reply:
I think that the `_utils` already says enough about the privacy of the module. Meaning that the functions themselves can just be public. That makes them easier to find in alphabetical order.